docs/source/examples/combine_techniques.md

```diff
@@ -0,0 +1,107 @@
+
++++
+
+## Setup
+
+We begin by importing the relevant modules and libraries required for the rest of this tutorial.
+
+```{code-cell} ipython3
+import cirq
+import numpy as np
+from mitiq.benchmarks import generate_rb_circuits
+from mitiq import MeasurementResult, Observable, PauliString, raw
+```
+
+## Task
+
+We will demonstrate using REM + ZNE on RB circuits, which are generated using Mitiq's built-in benchmarking function, `generate_rb_circuits`. More information on the RB protocol is available [here](https://learn.qiskit.org/course/quantum-hardware/randomized-benchmarking). In this example we use a two-qubit RB circuit with a Clifford depth (number of Clifford groups) of 5.
+
+```{code-cell} ipython3
+circuit = generate_rb_circuits(2, 5)[0]
+```
+
+## Noise model and executor
+
+The noise in this example is a combination of depolarizing and readout errors, the latter of which are modeled as bit flips immediately prior to measurement. We use an [`Executor`](../guide/executors.md) function to execute the quantum circuit with the noise model applied.
+
+```{code-cell} ipython3
+def execute(circuit: cirq.Circuit, noise_level: float = 0.005, p0: float = 0.05) -> MeasurementResult:
+    """Execute a circuit and return a ``MeasurementResult``, with depolarizing noise of strength
+    ``noise_level`` and readout errors with probability ``p0``."""
+    measurements = circuit[-1]
+    circuit =  circuit[:-1]
+    circuit = circuit.with_noise(cirq.depolarize(noise_level))
+    circuit.append(cirq.bit_flip(p0).on_each(circuit.all_qubits()))
+    circuit.append(measurements)
+
+    simulator = cirq.DensityMatrixSimulator()
+
+    result = simulator.run(circuit, repetitions=10000)
+    bitstrings = np.column_stack(list(result.measurements.values()))
+    return MeasurementResult(bitstrings)
+```
+
+## Observable
+
+We then obtain a unmitigated expectation value of the `ZI` + `ZZ` observable. In the ideal case the result is 2, but as we will see, the unmitigated result is impacted by depolarizing and readout errors.
+
+```{code-cell} ipython3
+obs = Observable(PauliString("ZI"), PauliString("IZ"))
+noisy = raw.execute(circuit, execute, obs)
+```
+
+```{code-cell} ipython3
+from functools import partial
+ideal = raw.execute(circuit, partial(execute, noise_level=0, p0=0), obs)
+f"Error without mitigation: {abs(ideal - noisy) :.5f}"
```

Comment:
Woah... I didn't know you could put strings/f-strings directly into the code cell like this and have them be executed and returned. Cool TIL. It does quote them in the return, however. 🤷

Reply:
Some of the other Mitiq examples have the f-strings, but I've changed this example to have `print` statements instead to avoid the weird quote marks.